Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIWebNavigation.mm

```diff
@@ -421,6 +421,56 @@
     [manager run];
 }
 
+TEST(WKWebExtensionAPIWebNavigation, ErrorOccurred)
+{
+    TestWebKitAPI::HTTPServer server({
+        { "/"_s, { { { "Content-Type"_s, "text/html"_s } }, "<iframe src='/frame.html'></iframe>"_s } },
+        { "/frame.html"_s, { HTTPResponse::TerminateConnection::Yes } },
+    }, TestWebKitAPI::HTTPServer::Protocol::Http);
+
+    auto *backgroundScript = Util::constructScript(@[
+        // Setup
+        @"function errorListener(details) {",
+        // A subframe should have been the one to have the error.
+        @"  browser.test.assertFalse(details.frameId == 0)",
```

Comment:
`browser.test.assertEq(details.frameId, 0)`

Reply:
This is `assertFalse` - I want them to _not_ be equal.

Reply:
Is there an `assertNeq`?

Reply:
Oh! Then `assertFalse` is fine.